projects/ngx-vcard/package.json

```
@@ -24,7 +24,7 @@
     "tslib": "2.3.1"
   },
   "peerDependencies": {
-    "@angular/common": ">=5.x <13.0.0",
-    "@angular/core": ">=5.x <13.0.0"
+    "@angular/common": ">=5.x <13.3.0",
```

As I think you have tested with the current 13.x release, would you please modify it to include all 13.x versions?
```suggestion
    "@angular/common": ">=5.x <14.0.0",
```